package com.auth.security.Exceptions.UserExceptions;

public class UsuarioNotFoundException extends RuntimeException{
    public UsuarioNotFoundException(long message) {
        super("No se encontr√≥ el usuario con ID: " +  message );
    }
}
